aptos-move/framework/src/lib.rs

```diff
@@ -1,7 +1,7 @@
 // Copyright (c) Aptos
 // SPDX-License-Identifier: Apache-2.0
 
-#![forbid(unsafe_code)]
+extern crate core;
```

Comment:
What and is this change for? :-/

I hope we're not using `unsafe` code because AFAIK there's no precedent for it yet in our codebase.